explorer/src/components/common/Identicon.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { useEffect, useRef } from "react";
+
+// @ts-ignore
+import Jazzicon from "jazzicon"; // TODO: Add definition file
+import bs58 from "bs58";
+import { PublicKey } from "@solana/web3.js";
+
+export function Identicon(props: {
+  address?: string | PublicKey;
+  style?: React.CSSProperties;
+  className?: string;
+}) {
+  const { style, className } = props;
+  const address =
+    typeof props.address === "string"
+      ? props.address
+      : props.address?.toBase58();
+  const ref = useRef<HTMLDivElement>();
```

Comment:
With this change you won't need to do `ref as any` below
```suggestion
  const ref = useRef<HTMLDivElement>(null);
```